functional/FunctionalTest.java

```diff
@@ -3115,15 +3115,18 @@ public static void composeObject_test1() throws Exception {
       List<ComposeSource> listSourceObjects = new ArrayList<ComposeSource>();
       listSourceObjects.add(s1);
       listSourceObjects.add(s2);
-
-      client.composeObject(bucketName, destinationObjectName, listSourceObjects, null, null);
-      Files.delete(Paths.get(filename1));
-      Files.delete(Paths.get(filename2));
-
-      client.removeObject(RemoveObjectArgs.builder().bucket(bucketName).object(filename1).build());
-      client.removeObject(RemoveObjectArgs.builder().bucket(bucketName).object(filename2).build());
-      client.removeObject(
-          RemoveObjectArgs.builder().bucket(bucketName).object(destinationObjectName).build());
+      try {
+        client.composeObject(bucketName, destinationObjectName, listSourceObjects, null, null);
```

Comment:
what will happen to the mint log here? We should not say success in this case, if there is a `not implemented` error returned we need to log it as N/A

Reply:
It wont say success. 
Flow is : 
**case A : If the composeObject is implemented**
1. It will create the object with name `destinationObjectName` 
2. Then it will delete the object `destinationObjectName`
3. The code in finally gets executed which deletes the source objects

**case B: If the composeObject is not  implemented**
1. The object wont be created 
2. Apart from `finally` no code below try gets called . So mint success will not be executed.
3. Finally block get executed always so the source object will deleted.